### PR TITLE
Enhancement of deploy doc

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Deploying the Arrebol Service
 
-The Arrebol service could be deployed as a docker container from a [arrebol docker image](https://hub.docker.com/repository/docker/ufcglsd/arrebol).  Most of the required dependencies to run Arrebol are kept in the own [Arrebol docker image](https://hub.docker.com/repository/docker/ufcglsd/arrebol). The only exception is the Docker Engine. To install it, in an apt-based environmnet, please run the `setup.sh` script as follow:
+The Arrebol service could be deployed as a docker container from a [arrebol docker image](https://hub.docker.com/repository/docker/ufcglsd/arrebol).  Most of the required dependencies to run Arrebol are kept in the own image. The only exception is the Docker Engine. To install it, in an apt-based environmnet, please run the `setup.sh` script as follow:
```
Comment (raonismaneoto) [inline deploy/README.md]: can be*

Comment (raonismaneoto) [inline deploy/README.md]: are kept in its own image

File: deploy/README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ All the configuration files are within the `deploy/config` directory. It is nece
 * The `postgres.env`, to define a password to the database;
 * The `pgadmin.env`, to define an the database admin credentials;
 * Assign the previously defined database password in the `spring.datasource.password` property on `application.properties` file;
-* Configure the `arrebol.json` file to tune Arrebol internals.
+* Configure the `arrebol.json` file to tune Arrebol internals. In it you will write the worker nodes ip.
```
Comment (raonismaneoto) [inline deploy/README.md]: You will write the worker nodes ip there

File: worker/deploy/README.md [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
+Arrebol tasks are processed by Workers. In a typical deployment, a few Workers are deployed together in a single virtual machine, a `Worker Node`. As each task run in an isolated docker container, the Docker engine and other dependencies must have be installed and configured in the Worker Node. Below, we described how to configure the Worker Node.
 
 ## Requeriments
 
-Before the configuration and installation of Worker Node dependencies, each Worker Node virtual machine should be configured to be reached via SSH (using a rsa key pair). Also, the [Ansible automation tool](https://www.ansible.com/) should be installed in the deploy coordination host.
+Before the configuration and installation of Worker Node dependencies, each Worker Node virtual machine should be configured to be reached via SSH (using a rsa key pair). Also, the [Ansible](https://www.ansible.com/) automation tool should be installed in the deploy `coordination host`.
```
Comment (raonismaneoto) [inline worker/deploy/README.md]: what is it,  deploy coordination host?

File: worker/deploy/README.md [hunk 4]
```diff
@@ -1,26 +1,31 @@
...
+To install Ansible run these commands:
+```bash
+sudo apt update
+sudo apt install software-properties-common
+sudo apt-add-repository --yes --update ppa:ansible/ansible
+sudo apt install -y ansible
+```
 
-`Warning:` Deployment does not work for nodes with Ubuntu version 14 or earlier.
+`Note: Deployment does not work for nodes with Ubuntu version 14 or earlier.`
```
Comment (raonismaneoto) [inline worker/deploy/README.md]: its ambiguous, you could rewrite it like: The deployment requires versions newer than 14 for ubuntu.

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: worker/deploy/README.md (hunk 3 above)
Comment (wesleymonte) [reply worker/deploy/README.md]: The host from which the commands are run.

File: deploy/README.md (hunk 1 above)
Comment (wesleymonte) [reply deploy/README.md]: done.

File: deploy/README.md (hunk 2 above)
Comment (wesleymonte) [reply deploy/README.md]: done.

File: worker/deploy/README.md (hunk 4 above)
Comment (wesleymonte) [reply worker/deploy/README.md]: done.

**Review by raonismaneoto — APPROVED** [review]

File: deploy/README.md (hunk 1 above)
Comment (wesleymonte) [reply deploy/README.md]: done.

**Review by thiagomanel — APPROVED** [review]
it seems better but it not good enough yet. I'll merge the doc but we need to improve it latter. For example, I missed an overview of the installation process. Also, which are the VM requirements?

**Pull request merged**